tests/Unit/CrudPanel/BaseDBCrudPanelTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Backpack\CRUD\Tests\Unit\CrudPanel;
+
+use Backpack\CRUD\CrudPanel;
+use Orchestra\Database\ConsoleServiceProvider;
+
+abstract class BaseDBCrudPanelTest extends BaseCrudPanelTest
+{
+    /**
+     * @var CrudPanel
+     */
+    protected $crudPanel;
```

Comment:
Aren't these two already here because of the parent class (I guess it doesn't hurt to redeclare them)?